socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -61,31 +58,59 @@ export default Vue.component("author-bar", {
         isUserAuthentificated: {type: Boolean, required: true},
     },
     data() {
-        return {showProfileBox: false}
+        return {
+            showProfileBox: false,
+            _isUserFollowingAuthor: this.isUserFollowingAuthor,
+        }
     },
     computed: {
+        isUserRemote(){
+            return !this.isUserLocal
+        },
         showFollowBtn() {
-            return this.isUserAuthentificated
-                    && !this.isUserLocal
-                    && !this.isUserFollowingAuthor
+            return this.canFollow && !this.$data._isUserFollowingAuthor
```

Comment:
> Yes. As told, earlier, the `_` prefixed data items work differently.

Actually, [it's written in the documentation](https://vuejs.org/v2/api/#data):

> Properties that start with `_` or `$` will not be proxied on the Vue instance because they may conflict with Vue’s internal properties and API methods. You will have to access them as `vm.$data._property`.